common/psbt_internal.c

```diff
@@ -23,10 +24,11 @@ psbt_input_set_final_witness_stack(const tal_t *ctx,
 }
 
 void psbt_finalize_input(const tal_t *ctx,
-			 struct wally_psbt_input *in,
+			 struct wally_psbt *psbt,
+			 size_t in,
```

Comment:
I had to change the signature of this function because libwally-core's new PSBT member accessor functions require a pointer to the PSBT.